manual/overview.wiki

```diff
@@ -45,6 +45,10 @@ For alternatives, see <<a_manual chapter="install" |Install>>.
       js_of_ocaml cubes.byte
 }}}
 
+JavaScript files generated by js_of_ocaml are UTF-8 encoded.
```

Comment:
Let's keep only that line. Again the two lines I suggested are going to create more confusion than needed I think.

Reply:
done